<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIOutput</class>
 <widget class="pDockWidget" name="UIOutput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>167</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/MessageBox.qrc">
    <normaloff>:/icons/taboutput.png</normaloff>:/icons/taboutput.png</iconset>
  </property>
  <property name="windowTitle">
   <string>Output</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout">
    <property name="margin">
     <number>5</number>
    </property>
    <property name="spacing">
     <number>3</number>
    </property>
    <item row="0" column="0" colspan="2">
     <widget class="QPlainTextEdit" name="tbOutput">
      <property name="undoRedoEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="tabStopWidth">
       <number>20</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pDockWidget</class>
   <extends>QDockWidget</extends>
   <header location="global">pDockWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/MessageBox.qrc"/>
 </resources>
 <connections/>
</ui>
